src/types/ErrorsMessages.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorsMessages {
+  Load = 'Unable to load a todos',
+  Post = "can't send post request",
```

Comment:
Consider using more descriptive error name. For example, instead of `Post`, use `UnableToSendPostRequest`. 📝